aws-athena/ctas/proximity-crosswalk_year_fill.sql

```diff
@@ -38,6 +38,8 @@ WITH unfilled AS (
             AS lake_michigan_data_year,
         MAX(dist_pin_to_major_road.nearest_major_road_data_year)
             AS nearest_major_road_data_year,
+        MAX(dist_pin_to_secondary_road.nearest_secondary_road_data_year)
+            AS nearest_secondary_road_data_year,
```

Comment:
**nitpick:** Technically the columns in here and the proximity table are in alphabetical order. Does it make sense to move this below, `nearest_park_data_year`?

Reply:
yep